### ISSUE TITLE
Icons for SCALE Mobile App

### ISSUE THREAD

**Issue by irabinovitch** [socallinuxexpo/SCALE-Design#169]
@hriday  has started working on the mobile app for SCALE this year. It's a mix of a badge scanning app and attendee guide. Anyways, the app developer is asking for the following assets.

- [ ]  App Icon 1024px by 1024px (used for app in play store, ideally just Troy?)
- [ ] Header Logo 300px by 60px  (must be transparent background)
- [ ] Splash Screen 480 x 320 (Loading page when app is starting)
- [ ] Splash Screen 640 x 960 (Loading page when app is starting)
- [ ] Splash Screen 1024 x 500 (Loading page when app is starting)
- [ ] Splash Screen 1242 x 2208 (Loading page when app is starting)
- [ ] Splash Screen 2048 x 1536 (Loading page when app is starting)

They're also letting us pick the colors for the app itself but we need to provide them in HEX:

- [ ] Primary Color: (Used for the header bar and link text HEX color e.g., #E53417)
- [ ] App Section Icon HEX Color (e.g., #E53417) *

**Comment by irabinovitch** [comment]
@ScislaC FYI.  I've also pinged a few of the graphics volunteers who came via the volunteer form on the off chance they have time.

**Comment by garethgreenaway** [comment]
@hriday is the code for this online somewhere?

**Comment by irabinovitch** [comment]
Not at this time.

**Comment by ScislaC** [comment]
@irabinovitch Thanks for the heads up. I have carved out some time tomorrow for some design time for a few high priority assets (things that need to be produced by third parties e.g. shirt, bag, & stickers) and will have these items in the queue after those things. BTW I did that additional banner ad size you requested a few days ago, but I need to tweak it more before submitting... that will likely be my warm up tomorrow.
